### PR TITLE
WR-1455 occupational interest filter drop down and tooltip

### PR DESCRIPTION
**Pull request by rjkuyvenhoven**
There are a few styling issues I haven't been able to resolve.

1. The Occupational Interest border is slightly taller than the other fields.  I changed the line height for all the fields. But this one is always slightly taller.
2. The drop down doesn't extend to the full width of the field.
3. The lighter blue focus border doesn't show up for the Occupational Interest field when selecting from the drop down.

### CODE REVIEW

File: src/web/themes/custom/workbc/workbc.theme [hunk 1]
```diff
@@ -208,3 +208,30 @@ function workbc_preprocess_menu(&$vars) {
...
   }
 }
+
+
+/**
+ * Implements hook theme_suggestions_paragraph_alter().
+ */
+function workbc_theme_suggestions_form_element_alter(array &$suggestions, array $variables) {
+  
+  if (substr($variables['element']['#id'], 0, 26) == "edit-occupational-interest") {
```
Comment (infojunkie) [inline src/web/themes/custom/workbc/workbc.theme:218]: This seems overly specific, for example against future versions of Views. Can we target a whole id, or something else that is stable in this element?

File: src/web/themes/custom/workbc/workbc.theme [hunk 2]
```diff
@@ -208,3 +208,30 @@ function workbc_preprocess_menu(&$vars) {
...
+
+/**
+ * Implements hook theme_preprocess_form_element().
+ */
+function workbc_preprocess_form_element(&$variables) {
+
+  if (substr($variables['element']['#id'], 0, 26) == "edit-occupational-interest") {
+    $text = \Drupal::config('workbc_custom.settings')->get('tooltips.occupationalinterest');
+    if (is_null($text) || !isset($text)) {
+      $text = 'Careers based on <a href="/plan-career/skills-future-workforce">six interest types</a>.';
```
Comment (infojunkie) [inline src/web/themes/custom/workbc/workbc.theme:232]: Please place this in the template directly - we don't need to parameterize it or even store it as config.

File: src/web/modules/custom/workbc_custom/src/Form/AdminSettingsForm.php [hunk 3]
```diff
@@ -68,6 +68,26 @@ public function buildForm(array $form, FormStateInterface $form_state) {
           '#format' => $text['format'],
         ];
 
+
+        $form['tooltips'] = [
```
Comment (infojunkie) [inline src/web/modules/custom/workbc_custom/src/Form/AdminSettingsForm.php:72]: We don't need to parameterize the tooltip. Let's keep it in the theme template.

**Review by infojunkie — CHANGES_REQUESTED** [review]
In general, let's remove the tooltip config and any associated code. We can hard-code this tooltip in the template.

File: src/web/themes/custom/workbc/workbc.theme (hunk 1 above)
Comment (rjkuyvenhoven) [reply src/web/themes/custom/workbc/workbc.theme]: I've modified this to use element['#name'] and didn't limit suggestions to 'occupation interest' only

**Comment by rjkuyvenhoven** [comment]
I've removed tooltip configuration and related code.

**Review by infojunkie — APPROVED** [review]
Lgtm, thanks. Please don't merge until I test locally and I take a stab at the remaining theming issues.

**Pull request merged**